scripts/publish/upload-to-s3.js

```diff
@@ -5,14 +5,18 @@ const { getName, getVersion } = require('../utils');
 
 const bucket = process.env.S3_BUCKET_NAME;
 const location = path.join(process.cwd(), 'lib', 'scripts');
+const gtmTemplate = location.includes('marketing-analytics-browser') ? `amplitude-gtm-min.js.gz` : ``;
 const files = ['amplitude-min.js.gz', 'amplitude-min.umd.js.gz'];
+if (gtmTemplate) files.push(gtmTemplate);
```

Comment:
`lib/scripts/amplitude-gtm-min.js.gz` won't exist in `analytics-browser`, and this upload to s3 file is shared for both analytics-browser and marketing-analytics-browser, so we need to exclude `amplitude-gtm-min.js.gz` when running upload to s3 for `analytics-browser` otherwise, all the job will be failed.